Pokemon Showdown Enhanced Tooltips/js/showPokemonTooltip.js

```diff
@@ -436,6 +436,29 @@ ShowdownEnhancedTooltip.BattleTypeChart = {
 	},
 };
 
+ShowdownEnhancedTooltip.getStatbarHTML = function getStatbarHTML(pokemon) {
+	let buf = '<div class="statbar' + (this.siden ? ' lstatbar' : ' rstatbar') + '" style="display: none">';
+	const ignoreNick = this.siden && (this.scene.battle.ignoreOpponent || this.scene.battle.ignoreNicks);
+	buf += `<strong><a href="https://www.smogon.com/dex/sm/pokemon/${pokemon.id}/" target="_blank" style="color: #222222; text-decoration: none;">${BattleLog.escapeHTML(ignoreNick ? pokemon.species : pokemon.name)}</a>`;
```

Comment:
This style stuff is optional:

```
style="color: #222222; text-decoration: none;"
```

But I felt that the default link style (blue and underlined) made the names stand out too much. This just ensures that they look the same as they did before the link was added.